docs/projects/fps/tutorial.md

```diff
@@ -404,94 +560,134 @@ namespace Fps
 ```
 Let’s break down what the above snippet does:
 
-* `[WorkerType(WorkerUtils.UnityGameLogic)]`<br>
-This `WorkerType` annotation decorates the class `HealthPickupServerBehaviour`. It tells SpatialOS to **only** enable this class on `UnityGameLogic` server-workers, ensuring that it will never run on your client-workers.
-* `[Require] private HealthPickupWriter healthPickupWriter;`<br>
-This is an instruction to the server-worker running the `HealthPickupServerBehaviour` class. It tells the server-worker to **only** enable this script on the worker with write-access to the `HealthPickup` component.
-* `private void OnTriggerEnter(Collider other)`<br>
-Most functions will **only** be called if the script component's `enabled` property is true, but `OnTriggerEnter` is called even when this is false. It is unusual in this sense. For this reason, scripts which use `OnTriggerEnter` **must** check whether the `Writer` is null (indicating a lack of authority) before using functions on the writer.
-* `private void SetIsActive(bool isActive)`<br>
-This function performs a component update. It sets the value of `IsActive`, the compoment property we [defined earlier](#define-a-new-spatialos-component), to a given value.
-* `healthPickupWriter?.SendUpdate(new HealthPickup.Update`<br>
-When sent to SpatialOS, the `HealthPickup.Update` object updates the `HealthPickup` component.
-* `private void HandleCollisionWithPlayer(GameObject player)`<br>
-This function will be called any time a player walks through a health pack. It handles cross-worker interaction using [commands](https://docs.improbable.io/reference/latest/shared/glossary#command). When you send a command it acts as a request, which SpatialOS delivers to the single worker that has write-access for the component that the command is intended for.<br>
-Cross-worker interactions can be necessary when your game has multiple `UnityGameLogic` server-workers, because the worker with write-access for the `HealthPack` entity may not be the same worker that has write-access to the `Player` entity who has collided with that health pack.<br>
-* `private IEnumerator RespawnCubeRoutine()`<br>
-This coroutine re-activates consumed health packs after a cool-down period. It starts at the end of the `HandleCollisionWithPlayer` function as well as in `OnEnable` for any health pack entities which are inactive. Running coroutines are stopped in `OnDisable`.
+* `[WorkerType(WorkerUtils.UnityGameLogic)]`<br/>
+This `WorkerType` annotation marks this `MonoBehaviours` to only be enabled for a specific worker-type. In this case, this `Monobehaviour` will only be enabled on `UnityGameLogic` client-workers, ensuring that it will never run on your server-workers.
```

Comment:
```suggestion
This `WorkerType` annotation marks this `MonoBehaviours` to only be enabled for a specific worker-type. In this case, this `MonoBehaviour` will only be enabled on `UnityGameLogic` client-workers, ensuring that it will never run on your server-workers.
```